Formula/fastfetch.rb

```diff
@@ -0,0 +1,32 @@
+class Fastfetch < Formula
+  desc "Like neofetch, but much faster because written in C"
+  homepage "https://github.com/LinusDierheimer/fastfetch"
+  url "https://github.com/LinusDierheimer/fastfetch/archive/refs/tags/1.7.2.tar.gz"
+  sha256 "f0778cdc6d33f5399bb82a8689b816ec3ff1d6b3eb74488e88d7a92a0bd3f7b6"
+  license "MIT"
+  head "https://github.com/LinusDierheimer/fastfetch.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "libplist" => :build
+  depends_on "pkg-config" => :build
+  depends_on "vulkan-loader" => :build
+  depends_on :macos
```

Comment:
Why not Linux?

Reply:
I dont have a linux machine to test and according to the official document fastfetch requires a lot dependencies to be useful.

However fastfetch should build without any dependencies. I will try to make it work on linux